kaolin/datasets/shapenet.py

```diff
@@ -126,15 +96,12 @@ class ShapeNet_Meshes(data.Dataset):
         train (bool): return the training set else the test set
```

Comment:
Return train set (if True), test set otherwise.

Reply:
This is actually not part of my PR, we intend to destroy this class to replace it by the one I've made.